haystack/nodes/answer_generator/openai.py

```diff
@@ -186,6 +195,23 @@ def __init__(
             logger.debug("Using GPT2TokenizerFast")
             self._hf_tokenizer: PreTrainedTokenizerFast = GPT2TokenizerFast.from_pretrained(tokenizer)
 
+    def using_azure(self) -> bool:
```

Comment:
Could we make this method private? Otherwise, it will appear in the API docs and I don't think it needs to.